bokehjs/src/coffee/models/selections/selection.ts

```diff
@@ -0,0 +1,57 @@
+import {Model} from "../../model"
+import * as p from "core/properties"
+import {union, intersection} from "core/util/array"
+import {merge} from "core/util/object"
+
+export class Selection extends Model {
+
+  indices: number[]
+  final: boolean
+  [key: string]: any
+
+  initialize(options: any): void {
+    super.initialize(options)
```

Comment:
Missing `attrs`. Compiler is actually complaining about this. This is such a common error across bokehjs (views take only `options` argument).